src/migrations/actions/202406081700_profile_exp.ts

```diff
@@ -0,0 +1,12 @@
+import Profile from '../../modules/profile/model';
+import type { QueryWithHelpers, UpdateWriteOpResult } from 'mongoose';
+
+export default {
+  async up(): Promise<QueryWithHelpers<UpdateWriteOpResult, unknown, unknown, unknown, 'updateMany'>> {
+    return Profile.updateMany({ exp: { $exists: true } }, { $set: { exp: 1 } });
+  },
+
+  async down(): Promise<void> {
+    await Profile.updateMany({}, { $unset: { exp: '' } });
```

Comment:
This is incorrect. You cannot remove exp completly of profile when migration fails